subt_ign/launch/competition.ign

```diff
@@ -300,7 +304,7 @@
     <record>
       <enabled>true</enabled>
       <!-- This path is used by cloudsim, please do not change -->
-      <path>/tmp/ign/logs/gazebo</path>
+      <path>/tmp/ign/logs</path>
```

Comment:
can you revert this change and the changes below to `static_update_frequency`?

Reply:
The static_update_frequency changes really need to be made, in this or another PR.  Ground truth is broken unless those are increased.

Reply:
let's try get the husky changes in first in this PR and review the other launch file changes separately.